Dockerfile.beagle

```diff
@@ -16,6 +14,7 @@ ARG JAR_PATH=/opt/java/bin
 COPY --from=builder /src/beagle.jar $JAR_PATH/beagle.jar
 RUN chmod 555 $JAR_PATH/beagle.jar
 
-WORKDIR /app
-
-ENTRYPOINT [ "java" ]
+# Create an entrypoint for the binary
+RUN echo "#!/bin/bash\njava -jar $JAR_PATH/beagle.jar \$@" > /entrypoint.sh && \
```

Comment:
I think, that java has an ENV variable that can be set, where to search for jars...